### PR TITLE
chore: reference Keep A Changelog and make it mandatory

### PR DESCRIPTION
**Pull request by carslen**
Be more precise that keep a changelog is mandatory for `CHANGELOG.md` file content.

### CODE REVIEW

**Review by FaGru3n — APPROVED** [review]
LGTM

**Comment by carslen** [comment]
@Siegfriedk please review/approve

File: docs/release/trg-1/trg-1-3.md [hunk 1]
```diff
@@ -2,17 +2,24 @@
...
 ## Why
 
-To track changes of releases a `CHANGELOG.md` must be added to repositories. The versioning itself
-follows [semantic versioning](https://semver.org/). For easy changelog creation [conventional committing](https://www.conventionalcommits.org/en/v1.0.0/#summary) is strongly recommended.
+To track changes of releases a `CHANGELOG.md` must be added to repositories. The format should follow the format
+explained at [Keep A Changelog](https://keepachangelog.com/en/1.0.0/).
 
-Recommendations for changelog format can be found [here](https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md).
+The versioning of releases must
+follows [semantic versioning](https://semver.org/). For easy changelog
```
Comment (Siegfriedk) [inline docs/release/trg-1/trg-1-3.md:21]: It reads slightly weird. Perhaps "The versioning of a release must follow?

**Review by Siegfriedk — APPROVED** [review]
see comment

File: docs/release/trg-1/trg-1-3.md (hunk 1 above)
Comment (carslen) [reply docs/release/trg-1/trg-1-3.md]: fixed typo

**Pull request merged**